tests/services/deduplication/test_dedup_embedder.py

```
@@ -48,22 +48,17 @@ def mock_vectorizer():
+
     mock_tfidf_cls = MagicMock(return_value=mock_vectorizer)
 
-    with patch.dict(
-        "sys.modules",
-        {
-            "sklearn": MagicMock(),
-            "sklearn.feature_extraction": MagicMock(),
-            "sklearn.feature_extraction.text": MagicMock(),
-        },
+    with (
+        patch("file_organizer.services.deduplication.embedder._SKLEARN_AVAILABLE", True),
```

Ruff (0.15.9)

[warning] Boolean positional value in function call

(FBT003)